File: library/src/main/java/org/angmarch/views/SimpleSpinnerTextFormatter.java
package org.angmarch.views;

import android.text.Spannable;
import android.text.SpannableString;

public class SimpleSpinnerTextFormatter<Object> implements SpinnerTextFormatter<Object> {

    @Override
    public Spannable formatSelectedText(Object text) {
        return new SpannableString(text.toString());
    }

    @Override
    public Spannable formatEntryText(Object text) {
        return new SpannableString(text.toString());
    }
}

File: library/src/main/java/org/angmarch/views/SimpleSpinnerTextFormatter.kt
package org.angmarch.views

import android.text.Spannable
import android.text.SpannableString

open class SimpleSpinnerTextFormatter : SpinnerTextFormatter<Any?> {
    override fun <T> format(item: T?): Spannable? {
        return SpannableString(item.toString())
    }

}